core/object/script_language_extension.h

```diff
@@ -655,13 +655,18 @@ VARIANT_ENUM_CAST(ScriptLanguageExtension::CodeCompletionLocation)
 
 class ScriptInstanceExtension : public ScriptInstance {
 public:
-	const GDExtensionScriptInstanceInfo2 *native_info;
+	const GDExtensionScriptInstanceInfo3 *native_info;
+
+#ifndef DISABLE_DEPRECATED
 	bool free_native_info = false;
 	struct {
 		GDExtensionScriptInstanceNotification notification_func = nullptr;
+		GDExtensionScriptInstanceFreePropertyList free_property_list_func = nullptr;
+		GDExtensionScriptInstanceFreeMethodList free_method_list_func = nullptr;
 	} deprecated_native_info;
 
 	GDExtensionScriptInstanceDataPtr instance = nullptr;
+#endif // DISABLED_DEPRECATED
```

Comment:
```suggestion
```

Reply:
Ah, thanks, good catch! Fixed in my latest push